packages/react-frontend/src/routes/FooterFiles/AboutUs.jsx

```diff
@@ -0,0 +1,37 @@
+import PropTypes from "prop-types";
+import Navbar from "../Navbar";
+//import "../../index.css";
+import Footer from "./Footer";
+import "./Footer.css"
+
+
+
+export default function About() {
+
+  return (
+    <>
+    <div className="loading-text">
+      <Navbar withLogo={true} />
+      <span className="about-us">
+      <h2>About Us: Our Story</h2>
+      </span>
+      <span className="about-us">
+      
+      <p>Once upon a time in the bustling city of Melodyville, a group of five quirky friends—Tammy, Braeden, Sara, Yasemin, and Sharon—set out on a musical adventure that would change their lives forever. Tammy, with her wild purple hair and endless energy, was a music aficionado who could identify a song by its first note. Braeden, the tech wizard with an impressive collection of vintage vinyl records, dreamed of a digital platform where music lovers could share their eclectic tastes. Sara, a former child prodigy turned DJ, brought her beat-dropping skills and infectious laughter to the group. Yasemin, a poetic soul with a knack for penning song lyrics, added the lyrical magic to their team. And then there was Sharon, the ultimate social media guru, whose witty posts and memes could make anything go viral.</p>
```

Comment:
looks like the p tags are causing the errors with the linter, is it missing quotes?

Reply:
Yes, I noticed that. I will fix the linter error soon. I'm going to try a few different things to get them to work.

Reply:
Update: I fixed it!